[Stripe]: Add New Commands

farisaziz12 opened the pull request:
## Description

Added new commands to the current Stripe Quicklinks extension, which uses the Stripe API to give quick access to data for:
- Balance
- Balance Transactions
- Charges
- Connected Accounts
- Payment Intents
- Events

The Test/Live environments can be toggled. 

## Screencast

![stripe-extension-demo](https://user-images.githubusercontent.com/53216647/183179558-90a04d32-0572-4cd8-a8a0-e2414b884f20.gif)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

RSO reviewed (commented):
Looks good to me! I'll defer to the review team to make sure this grid-as-dashboard pattern is something we like, but functionality-wise this works as expected 😄 thanks!

RSO commented:
Ok, I checked in with @mathieudutour and @sxn and we think it'd better fit Raycast's design guidelines if we go with separate commands instead of this single command with a grid dashboard. We also think it makes sense to merge this extension with the pre-existing `stripe` extension, which is really a bunch of Quicklinks. This should leave us with a single, awesome Stripe Extension to rule them all!

RSO reviewed (changes requested):
Forgot to request changes 😄

farisaziz12 commented on extensions/stripe-dashboard/package.json:
```diff
@@ -0,0 +1,63 @@
+    {
+      "name": "stripeTestApiKey",
+      "type": "password",
+      "required": true,
```
It is required because all requests in the test env will fail without. I can make this optional and add error handling when it's undefined and a test env request is being made.

farisaziz12 replied:
I have now made both the keys options to allow users to install the extension and only use the quick links. When using the commands that hit the Stripe API there is additional error handling to prompt the user to add the appropriate keys for the environment they want data from.

farisaziz12 commented:
Hey @RSO 👋, 

All my work has been merged into the current Stripe extension in the format of separate commands. I have also added a changelog with the latest information, updated the metadata screenshots and provided a new GIF of the current extension's functionality. Feel free to give it a smoke test on your end and let me know if you spot anything else that can help improve the commands.

pradeepb28 commented:
Awesome work @farisaziz12 I was planning to add Stripe APIs glad to see you were super fast  🚀

RSO reviewed (approved):
This is awesome! 👏  Thank you for the changes!
